src/OSPSuite.UI/Views/Importer/ImportConfirmationView.cs

```diff
@@ -104,7 +100,15 @@ private void onDataSetNameSelected(object sender, EventArgs eventArgs)
 
          if (listBox.SelectedValue == null) return;  //here if null we have to empty the GUI
 
-         _presenter.DataSetSelected(listBox.SelectedValue.ToString(), listBox.SelectedIndex );
+         _presenter.DataSetSelected(listBox.SelectedValue.ToString(), listBox.SelectedIndex);
+      }
+
+      public override void InitializeResources()
+      {
+         base.InitializeResources();
+         //TODO CAPTION
+         Caption = "Confirmation";
```

Comment:
Make sure all capations are defined in constant and not in designer please. I put TODO everywhere that I find but there are other places